autosubmitconfigparser/config/configcommon.py

```diff
@@ -1671,7 +1672,8 @@ def load_last_run(self):
                 # Load data from last run
                 if (Path(self.metadata_folder) / "experiment_data.yml").exists():
                     with open(Path(self.metadata_folder) / "experiment_data.yml", 'r') as stream:
-                        self.last_experiment_data = yaml.load(stream, Loader=yaml.SafeLoader)
+                        yaml_loader = YAML(typ='safe')
+                        self.last_experiment_data = yaml_loader.load(stream)
```

Comment:
Is it only `yaml.load` that's affected by the update? We also have `yaml.dump` in this file… although I am not sure if it's used by any test.

Reply:
I've fixed the dump function, I'm not using "safe" at the moment of dumping because the exported file loses readability while the default mode is similar to what we have. The difference is that the order of the keys exported is different however that shouldn't matter

I've also added a test for the save function

Reply:
>I've fixed the dump function, I'm not using "safe" at the moment of dumping because the exported file loses readability while the default mode is similar to what we have. The difference is that the order of the keys exported is different however that shouldn't matter

Maybe this could be added as a comment near that part of the code. Sounds useful (and at least I'll very likely forget it soon :grimacing: )